CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 4.0.0-rc
+- Streaming support
+- 3rd party auth support
+- HTTP/2 support for .net standard 2.1 and above
```

Comment:
List all of the missing functions that got added?

Reply:
added